src/mobile-ui-react/MobileUi.tsx

```diff
@@ -4,14 +4,37 @@
 *--------------------------------------------------------------------------------------------*/
 import * as React from "react";
 import { BackendError, Localization } from "@itwin/core-common";
-import { getCssVariable, getCssVariableAsNumber } from "@itwin/core-react";
 import { ColorTheme, SessionStateActionId, SyncUiEventDispatcher, SyncUiEventId, SYSTEM_PREFERRED_COLOR_THEME, UiFramework, UiSyncEventArgs } from "@itwin/appui-react";
 import { EmphasizeElements, IModelApp, IModelConnection, ScreenViewport, SelectionSet, Tool, Viewport } from "@itwin/core-frontend";
 import { BeEvent, BeUiEvent, BriefcaseStatus, Id64Set, Listener } from "@itwin/core-bentley";
 import { getAllViewports, getEmphasizeElements, Messenger, MobileCore, UIError } from "@itwin/mobile-sdk-core";
 
 import "./MobileUi.scss";
 
+/** Props used by components that expect class name to be passed in.
+ *
+ * __Note__: Copied from @itwin/core-react, where it is being deprecated. It will **not ever** be
+ * deprecated from @itwin/mobile-ui-react.
+ * @public
+ */
+export interface ClassNameProps {
```

Comment:
lets review with the appui team why this was deprecated, and what they expected consumers to do, before we re-export a deprecated type from another repo as public here

Reply:
I changed this to draft for now while we decide what to do and created a new PR to allow lint to pass.